### DISCUSSION TITLE
Migrate shorturls

### DISCUSSION THREAD

**Discussion by mart-e** [getpelican/pelican#3120] [Q&A]
Hello,

When migrating my Wordpress website, I had two URLs to access an article:

- https://example.com/post/2023/04/17/my-super-article
- https://example.com/?p=42

I have managed to recover the first one by using

```
Save_as: /post/2023/04/17/my-super-article.html
Url: /post/2023/04/17/my-super-article
```
but I was wondering how I could get the second one. Is it possible to set multiple URL values? Or to create redirections in the definition of the article?

Thanks

**Comment by lioman** [comment]
You have different possibilities:

- adding all that information to your server and redirect/rewrite those urls
- using a plugin to achieve that (there is `pelican-alias` and `pelican-redirect` - but I haven't used them)

**Reply by mart-e** [reply]
oh cool, will give a go to the extensions, thanks
